src/ocamlorg_frontend/pages/community.eml

```diff
@@ -163,6 +165,34 @@ Layout.render
       </div>
   </div>
 </div>
+<div class="bg-default dark:bg-dark-default py-0 pt-20 md:pt-10 lg:p-20 pb-10 md:pb-20">
+  <div class="container-fluid">
+      <div class="text-center">
+        <h2 class="font-bold">Upcoming Events</h2>
+      </div>
+      <div class="grid lg:grid-cols-3 gap-8 lg:gap-12 mt-12">
+          <% upcoming_events |> List.iter (fun (event : Data.Event.t) -> %>
+          <a href="<%s event.url %>" class="border border-gray-200 p-8 rounded-xl transition-transform hover:scale-105">
+              <div class="font-semibold">
+                  <%s event.title %>
+              </div>
+              <div class="mt-4">
+                  <div class="flex items-center space-x-2">
+                      <%s! Icons.map_pin "h-5 w-5" %>
+                      <div>
+                          <%s event.textual_location %>
+                      </div>
+                  </div>
+                  <div class="flex items-center space-x-2 mt-2">
+                    <%s! Icons.calendar "h-5 w-5" %>
+                    <div><%s event.starts %></div>
+                  </div>
+              </div>
+          </a>
+          <% ); %>
+      </div>
+  </div>
+</div>
```

Comment:
This should be displayed only when there is a non-zero number of Events to show, i.e. when `List.length events > 0`.

You can do this either by adding an `if-then-else` block here, or you can create a sub-template `upcoming_events`.

Reply:
Thanks @sabine. I'm on it